packages/driver/src/cypress/runner.js

```diff
@@ -962,6 +972,10 @@ const create = function (specWindow, mocha, Cypress, cy) {
           break
       }
 
+      if (hasTestAlreadyRun(test)) {
+        return args[0]()
```

Comment:
what is `args[0]()` also? Have to look into another file to even know what this args is and even then gives no indication on what the first args is.